### PR TITLE
fix #2555 fix #2533 incorrect connection state

### LINKED ISSUE #2533: "You are disconnected and checked in"
## Issue - Bug

User reported during retro highlighting their avatar told them `You are disconnected and checked in` even though their light was green. This looks like a typo/state rendering bug

See the following screencap from the user:

image

- **OS:** macOS
- **Browser:** Chrome
- **Estimated effort:** 10.5 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### Acceptance Criteria (optional)
Users can:
   - Copy should read `You are connected and checked in` when their connection status is green

### LINKED ISSUE #2555: persist self connected state in relay
AC
- [ ] when a user loses internet connectivity, their dot goes gray
- [ ] clicking the menu shows "disconnected"
- [ ] reconnecting turns it green/connected again

effort: 10.5

### PR DESCRIPTION
**Pull request by mattkrick**
TEST
- [ ] on a team dahsboard, see that your dot is green, then quit the server, see that it goes gray
- [ ] repeat above step in retro lobby

fixes #2555

Note: I didn't touch the logic in action meetings since it's old, difficult, and works differently. we'll get this functionality for free when we refactor it to the new meeting format

### CODE REVIEW

**Comment by mattkrick** [comment]
@jordanh ready for review

File: src/universal/components/SocketHealthMonitor.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import {Component} from 'react'
 import withAtmosphere from 'universal/decorators/withAtmosphere/withAtmosphere'
 import popUpgradeAppToast from 'universal/mutations/toasts/popUpgradeAppToast'
 import {APP_VERSION_KEY} from 'universal/utils/constants'
+import {commitLocalUpdate} from 'react-relay'
 
 class SocketHealthMonitor extends Component {
```
Comment (jordanh) [inline src/universal/components/SocketHealthMonitor.js]: This logic reads simply enough

**Comment by jordanh** [comment]
# Test Cycle 1

## Tests

- [ ] on a team dahsboard, see that your dot is green, then quit the server, see that it goes gray
- [ ] repeat above step in retro lobby

## Notes

```
WARNING in [at-loader] ./src/universal/components/OrgAvatarInput.tsx:1:25
    TS2306: File '/Users/jordanhusney/Source/github/Parabol/action.git/node_modules/@mattkrick/sanitize-svg/dist/types/sanitizeSVG.d.ts' is not a module.

WARNING in [at-loader] ./src/universal/components/UserAvatarInput.tsx:1:25
    TS2306: File '/Users/jordanhusney/Source/github/Parabol/action.git/node_modules/@mattkrick/sanitize-svg/dist/types/sanitizeSVG.d.ts' is not a module.
```

**Comment by mattkrick** [comment]
fixed!

**Comment by jordanh** [comment]
# Test Cycle 2

## Tests

- [ ] on a team dashboard, see that your dot is green, then quit the server, see that it goes gray
- [ ] repeat above step in retro lobby

## Notes

**Comment by mattkrick** [comment]
> When User A and User B are on the team dash/retro lobby and User B closes their browser, User A does not see User B's dot go gray. Is this working as designed?

we persist connection state in the DB, so when testing locally, you'll need to flush it after every server crash. `yarn postdeploy` should fix that

**Pull request merged**